src/interface/storage.thrift

```diff
@@ -188,6 +208,13 @@ struct EdgePropRequest {
     5: list<PropDef> return_columns,
 }
 
+struct IndexScanRequest {
+    1: common.GraphSpaceID       space_id,
+    2: list<common.PartitionID>  parts,
+    3: common.IndexHint          hint,
+    4: list<string>              return_columns,
```

Comment:
Each element is the column name?

Reply:
Yes , each element is a column name. the column list corresponds to yield clause.